src/storybook/story-to-test/write-tests-file.ts

```diff
@@ -0,0 +1,38 @@
+import path from "path";
+import fs from "fs-extra";
+import { getAbsoluteFilePath as getStoryRunnerAbsoluteFilePath } from "../story-test-runner";
+import { StorybookStoryExtended } from "../get-stories";
+
+export type HermioneOpts = {
```

Comment:
Those options are passed and fixed during `INIT`

Others can be changed between test launches from gui